admin/jobs/river/org_jobs.go

```diff
@@ -125,7 +115,7 @@ func (w *StartTrialWorker) Work(ctx context.Context, job *river.Job[StartTrialAr
 		TrialEndDate: sub.TrialEndDate,
 	})
 	if err != nil {
-		w.logger.Error("failed to send trial started email", zap.String("org_name", trialOrg.Name), zap.String("org_id", trialOrg.ID), zap.String("billing_email", trialOrg.BillingEmail), zap.Error(err))
+		return fmt.Errorf("failed to send trial started email for organization %s: %w", trialOrg.Name, err)
```

Comment:
I wouldn't return an error here as job will try to start the trial again, although currently `w.admin.StartTrial` is idempotent but I would not bet on it on future. I think we should just get an alert when this happens.